apps/examples/testcase/le_tc/network/tc_net_poll.c

```diff
@@ -0,0 +1,183 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/*  @file tc_net_poll.c
+ * @brief Test Case Example for lwip_poll() API
+ */
+
+#include <stdio.h>
+#include <poll.h>
+#include <sys/ioctl.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <netutils/netlib.h>
+#include <lwip/sockets.h>
+#include <tinyara/os_api_test_drv.h>
+#include "tc_internal.h"
+
+/**
+* @testcase             :tc_net_poll_teardown_p
+* @brief                :check for poll teardown
+* @scenario             :
+* @apicovered           :lwip_poll()
+* @precondition         :
+* @postcondition        :
+*/
+
+static void tc_net_poll_teardown_p(int fd, int sd, struct poll_test_args *args)
+{
+	int ret = -1;
+	ret = ioctl(fd, TESTIOC_NET_POLL, args);
+	TC_ASSERT_GEQ_CLEANUP("lwip_poll", ret, 0, close(sd));
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase             :tc_net_poll_p
+* @brief                :check for poll setup
+* @scenario             :
+* @apicovered           :lwip_poll()
+* @precondition         :
+* @postcondition        :
+*/
+
+static void tc_net_poll_p(int fd)
+{
+	int ret = -1;
+	int sd = -1;
+	struct pollfd fds[1];
+	struct poll_test_args args;
+
+	sd = socket(AF_INET, SOCK_DGRAM, 0);
+
+	fds[0].fd = sd;
+	fds[0].events = POLLIN;
+
+	args.fd = sd;
+	args.fds = &fds;
```

Comment:
can you type case here to remove the warning messages

Reply:
corrected now.